Utility.cpp

```diff
@@ -22,7 +23,7 @@ std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num)
 {
     std::vector<std::unique_ptr<Item>> items;
     
-    while( num-- >= 0 )
+    while( num-- > 0 )
```

Comment:
This code did not need to be changed
- [ ] revert

Reply:
I left the same because when I request X number of items in the function call within each character instance constructor, it returns X+1 number of items